sync2jira/upstream_issue.py

```diff
@@ -252,26 +305,30 @@ def github_issues(upstream, config):
         if issue.get('milestone', None):
             issue['milestone'] = issue['milestone']['title']
 
-        if not issue.get('storypoints', None):
-            issue['storypoints'] = ''
-            orgname, reponame = upstream.rsplit('/', 1)
-            issuenumber = issue['number']
-            default_github_project_fields = config['sync2jira']['default_github_project_fields']
-            project_github_project_fields = config['sync2jira']['map']['github'][upstream]['github_project_fields']
-            github_project_fields = default_github_project_fields | project_github_project_fields
-            variables = {"orgname": orgname, "reponame": reponame, "issuenumber": issuenumber}
-            for fieldname, values in github_project_fields.items():
-                ghfieldname, _ = values
-                variables['ghfieldname'] = ghfieldname
-                response = requests.post(graphqlurl, headers=headers, json={"query": project_items_query, "variables": variables})
-                data = response.json()
-                if fieldname == 'storypoints':
-                    try:
-                        issue[fieldname] = data['data']['repository']['issue']['projectItems']['nodes'][0]['fieldValueByName']['number']
-                    except (TypeError, KeyError) as err:
-                        log.debug("Error fetching %s!r from GitHub %s/%s#%s: %s",
-                            ghfieldname, orgname, reponame, issuenumber, err)
-                        continue
+        issue['storypoints'] = None
+        issue['priority'] = ''
+        orgname, reponame = upstream.rsplit('/', 1)
+        issuenumber = issue['number']
+        default_github_project_fields = config['sync2jira']['default_github_project_fields']
+        project_github_project_fields = config['sync2jira']['map']['github'][upstream]['github_project_fields']
+        github_project_fields = default_github_project_fields | project_github_project_fields
+        variables = {"orgname": orgname, "reponame": reponame, "issuenumber": issuenumber}
+        for fieldname, values in github_project_fields.items():
+            response = requests.post(graphqlurl, headers=headers, json={"query": ghquery, "variables": variables})
+            data = response.json()
+            try:
+                nodes = data['data']['repository']['issue']['projectItems']['nodes'][0]['fieldValues']['nodes']
+                for item in nodes:
+                    if item.get('fieldName') and item.get('fieldName').get('name'):
+                        ghFieldName = item.get('fieldName').get('name')
```

Comment:
The first condition in line 322 should be `'fieldName' in item`, and the second condition you reuse on the next line (and you don't need the first `get()` call in it), so I suggest:
```suggestion
                    if 'fieldName' in item:
                        ghFieldName = item['fieldName'].get('name')
                        if ghFieldName:
```
or
```suggestion
                    ghFieldName = item.get('fieldName', {}).get('name')
                    if ghFieldName:
```